scripts/replicatePrivateErddapDeployments.py

```diff
@@ -148,27 +151,47 @@ def load_deployments():
     fail_counter = 5
     # try to release semaphore before attempting to get the response
     async with sem:
-        async with aiohttp.ClientSession() as session:
-            while True:
-                try:
-                    # This causes timeouts even when the max simultaneous connection
-                    # lock isn't released
-                    async with session.get(url) as response:
-                        with open(path_arg, 'wb') as f_handle:
-                            while True:
-                                chunk = await response.content.read(1024)
-                                if not chunk:
-                                    break
-                                f_handle.write(chunk)
-                        return await response.release()
-                except Exception as e:
-                    fail_counter -= 1
-                    log.exception("Failed to get %s", deployment)
-                    log.info("Attempts remainging: %s", fail_counter)
-                    if fail_counter <= 0:
-                        break
-                    else:
-                        break
+        try:
+            async with aiohttp.ClientSession(raise_for_status=True) as session:
+                while True:
+                    try:
+                        # This causes timeouts even when the max simultaneous connection
+                        # lock isn't released
+                        async with session.get(url) as response:
+                            with open(path_arg + '.tmp', 'wb') as f_handle:
+                                while True:
+                                    chunk = await response.content.read(1024)
+                                    if not chunk:
+                                        break
+                                    f_handle.write(chunk)
+                            return await response.release()
+                    except Exception as e:
+                        fail_counter -= 1
+                        log.exception("Failed to get %s", deployment)
+                        log.info("Attempts remainging: %s", fail_counter)
+                        if fail_counter <= 0:
+                            break
+                        else:
+                            break
```

Comment:
can just use a break here right? No if else

Reply:
else shouldn't be there